hgtv/views/video.py

```diff
@@ -39,21 +39,25 @@ def process_video(video, new=False):
         if parsed.netloc in ['youtube.com', 'www.youtube.com']:
             try:
                 video_id = parse_qs(parsed.query)['v'][0]
-                r = requests.get('https://gdata.youtube.com/feeds/api/videos/%s?v=2&alt=json' % video_id)
-                jsondata = r.json() if callable(r.json) else r.json
-                if jsondata is None:
+                r = requests.get('https://www.googleapis.com/youtube/v3/videos?part=snippet&id={video_id}&key={api_key}'.format(
+                    video_id=video_id, api_key=app.config.get('YOUTUBE_API_KEY', '')
+                ))
+                try:
+                    jsondata = r.json() if callable(r.json) else r.json
```

Comment:
Use `jsondata = r.json()`. That callable check was because Requests changed their API many years ago and we did not have minimum dependency requirement in our code at that time. We no longer need to support older versions of Requests. We can remove the callable check across HGTV.